databags/translate.ini

```diff
@@ -122,7 +122,7 @@ spanish = Español
 french = Français
 korean = 한국어
 portuguese = Português
-chinese = 漢語
+chinese = 中文
```

Comment:
Let me explain, in my personal opinion, "中文" and "漢語" express the same thing, with "中文" being the more common usage and "漢語" only appearing in history, etc., and from my observation, translators usually choose "中文" over "漢語" even if it's only in simplified Chinese, so I'm changing it to "中文" here.

Since the previous translator's translation was in traditional Chinese, I subconsciously translated it that way, which I didn't realise, sorry. Then, as you said, we really should split Chinese into zh_TW and zh_CN. I'll finish that part in the next few days.

Reply:
Would `中文 (汉语)` and `中文 (漢語)` be an appropriate translation for "Chinese (Simplified)" and "Chinese (Traditional)"? (I'm setting up the translation for the tutorial, and I need to include a link there as well)

Reply:
Usually, traditional Chinese and simplified Chinese correspond to "中文(繁體)" and "中文(简体)", or you can just use "繁體中文" and "简体中文", which are also widely accepted.